### ISSUE TITLE
Breakpoint does not hit as expected in web.

### ISSUE THREAD

**Issue by safield** [Dart-Code/Dart-Code#3931]
I have created a brand new flutter project.

I have changed my main.dart as follows...

```
void main() async {
  print("blah"); // I put breakpoint here
}
```

I have added a breakpoint at the commented line.

When I run/debug this application, the breakpoint does not hit where expected. Instead a breakpoint is trigger at the following line/file...

web_entrypoint.dart
```
// @dart=2.16
// Flutter web bootstrap script for package:web_breakpoint_test/main.dart.

import 'dart:ui' as ui;
import 'dart:async';

import 'package:web_breakpoint_test/main.dart' as entrypoint;

typedef _UnaryFunction = dynamic Function(List<String> args);
typedef _NullaryFunction = dynamic Function();
Future<void> main() async {
  await ui.webOnlyInitializePlatform(); // BREAK POINT TRIGGERS HERE
  if (entrypoint.main is _UnaryFunction) {
    return (entrypoint.main as _UnaryFunction)(<String>[]);
  }
  return (entrypoint.main as _NullaryFunction)();
}
```

If I remove the breakpoint from main.dart, no breakpoint fires, if I add the breakpoint in, the web_entrypoint.dart is opened and the breakpoint fires at the commented line.

When I press continue, the print statement (where the breakpoint is placed) does print out to the log as expected.

**Comment by DanTup** [comment]
This sounds like https://github.com/dart-lang/webdev/issues/1557. There was a change in Chrome v100 that removed some data that dwds (a Dart debug proxy used by Flutter to debug JS as if it was Dart running in the Dart VM) was using.

Please follow https://github.com/dart-lang/webdev/issues/1557 for updates. Thanks!

**Comment by DanTup** [comment]
My understanding is that this issue has been fixed for the Chrome device in Flutter 2.10.5 (see https://github.com/flutter/flutter/wiki/Hotfixes-to-the-Stable-Channel#2105-april-18-2022), though there's still some outstanding work to fix it for the Web Server device.
